### DISCUSSION TITLE
CT101 Thermostat  not reporting humidity

### DISCUSSION THREAD

**Discussion by pailloM** [zwave-js/zwave-js#7216] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
Thermostat is not reporting humidity
Seems to have happened before #4643 but it is back...

What did you expect to happen instead?
Report humidity

1 Add Thermostat as a new Zwave node
2 Check Multi Level Sensor V2 in ZWave2MQTT
3) Only Air Temp... No Humidity report.

Anything else we should know?
zwavejs-ui debug info with original config, no humidity:
[zwavejs-ui_debug_info_original.json](https://github.com/user-attachments/files/17239180/zwavejs-ui_debug_info_original.json)

zwavejs-ui debug info with updated config reporting humidity:
[zwavejs-ui_debug_info.json](https://github.com/user-attachments/files/17239182/zwavejs-ui_debug_info.json)

device config updated reporting humidity:
[ct101.json](https://github.com/user-attachments/files/17239181/ct101.json)

### Software versions

zwave-js-ui: 9.21.1.e8667de
zwave-js: 13.4.0

### Z-Wave Controller (Stick/Dongle/...)

 AEON Labs 	Z‐Stick Gen5 USB Controller 	ZW090 

### Device information

Radio Thermostat Company of America (RTC) 
Z-Wave Thermostat 	CT101
Node 044 

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[z-ui_2024-10-03.log](https://github.com/user-attachments/files/17239198/z-ui_2024-10-03.log)

[a comment by an automated account is not shown]

**Reply by AlCalzone** [reply]
What the bot said

**Comment by pailloM** [comment]
I followed the instructions:
https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs

This one doesn't seem as complete as the other one:
[zwavejs_2024-10-03.log](https://github.com/user-attachments/files/17252678/zwavejs_2024-10-03.log)

**Reply by AlCalzone** [reply]
> This one doesn't seem as complete as the other one:

This one is not on loglevel "debug"

**Comment by pailloM** [comment]
Hopefully this one has the right level of logs. It includes an interview of the thermostat. It is now Node 046.

[zwavejs_2024-10-09.log](https://github.com/user-attachments/files/17320749/zwavejs_2024-10-09.log)

**Reply by AlCalzone** [reply]
> Hopefully this one has the right level of logs

Still not "debug". Did you save after changing the settings?

> It is now Node 046.

Just FYI, you don't have to exclude and re-include the device to re-interview it:
![grafik](https://github.com/user-attachments/assets/a380a88e-0c5b-4722-b483-ede5ada9455e)

**Comment by pailloM** [comment]
Trying again, now Node 047. The thermostat doesn't go to sleep anymore. So I tried to exclude/include it a few times but it still is not going to sleep, not why that is yet.

The settings are definitely saved. Zwavejs-ui show logs are debug level, I am not sure why the driver logs are not.
[zwavejs_2024-10-13.log](https://github.com/user-attachments/files/17359148/zwavejs_2024-10-13.log)

**Reply by AlCalzone** [reply]
And you're sure you edited the loglevel for the driver logs (under Z-Wave) and not the application logs (under General)?

**Comment by pailloM** [comment]
That was the trick, I missed that the logs configs are in two different places! Sorry for this.
This one has got the node interview - Node 047
[zwavejs_2024-10-14.log](https://github.com/user-attachments/files/17372271/zwavejs_2024-10-14.log)

I have also noticed the thermostat time is not set and the screen is always on
